tests/validators/tests.py

```diff
@@ -219,6 +219,7 @@
     # Trailing junk does not take forever to reject
     (URLValidator(), 'http://www.asdasdasdasdsadfm.com.br ', ValidationError),
     (URLValidator(), 'http://www.asdasdasdasdsadfm.com.br z', ValidationError),
+    (URLValidator(schemes='__all__'), 'foo://hello.com', None),
```

Comment:
How about some tests for `schemes='__all__'` and URLs with schemes that don't match the regular expression?